lib/Bio/EnsEMBL/DataCheck/Checks/CheckMSANames.pm

```diff
@@ -0,0 +1,78 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckMSANames;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckMSANames',
+  DESCRIPTION    => 'Ensure that every MSA method has a name in species_set_header',
+  GROUPS         => ['compara', 'compara_multiple_alignments'],
+  DATACHECK_TYPE => 'critical',
+  DB_TYPES       => ['compara'],
+  TABLES         => ['method_link', 'method_link_species_set', 'species_set_header']
+};
+
+sub skip_tests {
+    my ($self) = @_;
+    my $mlss_adap = $self->dba->get_MethodLinkSpeciesSetAdaptor;
+    my @methods = qw( EPO EPO_LOW_COVERAGE PECAN );
+    my $db_name = $self->dba->dbc->dbname;
+    
+    my @mlsses;
+    foreach my $method ( @methods ) {
+      my $mlss = $mlss_adap->fetch_all_by_method_link_type($method);
+      push @mlsses, @$mlss;
+    }
+
+    if ( scalar(@mlsses) == 0 ) {
+      return( 1, "There are no multiple alignments in $db_name" );
+    }
+
+}
```

Comment:
The HCs follow `method_link.class` instead of the type itself, so that they can work regardless of the names. It makes the code future-proof against new methods being added, or methods being renamed, cf https://www.ebi.ac.uk/panda/jira/browse/ENSINT-334

Reply:
`skip_tests` starts being used in many modules. We'll have to see if it can be factored out

Reply:
Are these comments about using the method_link.class rather than method_link.type to be addressed in this PR? Yes this code is not future proof, but there are also some benefits to using the method_link.types in the DCs, for example, it splits the tests into more manageable chunks, it also makes debugging easy if a pattern is spotted in a particular method_link.type. If a new method_link.type is added, currently it will just need to be added to the array of method_link.types to check. Is there an API method to retrieve the method_link.class otherwise? Or should there be if there isn't? (I couldn't find one). Alternatively, I could revert back to the SQL queries like in the HCs? Which is your preferred option?

I am not sure what to do about the `skip_tests`, but I suppose it will be factored out once every division is as thoroughly compared as vertebrates...

Reply:
No I don't think the comments about using `method_link.class` should be addressed in this PR. I was planning on creating tickets to review 1) the usage of type vs class and 2) skip_tests, once all the DCs have been merged in.

We have `fetch_all_by_class_pattern` in MLSSAdaptor, but let's see when we work on 1) if that's convenient or not.

In the meantime, just make sure that you have in the DCs all the types that match the class patterns used in the HCs